packages/pwa-kit-runtime/src/utils/ssr-namespace-paths.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * This file defines the /mobify paths used to set up our Express endpoints.
+ *
+ * If a namespace for the /mobify paths is defined, the methods in here will return the
+ * namespaced path. ie. /namespace/mobify/...
+ */
+
+const MOBIFY_PATH = '/mobify'
```

Comment:
Have we thought about making the default namespace `mobify`.. what way when people want to change the namespace getter they still don't end up with a path that has "mobify" in it?

Reply:
Update: mobify is required as per mobify cloud api